### PR TITLE
Fix/external path

### PR DESCRIPTION
**Pull request by alex-Arc**

### CODE REVIEW

File: apps/server/src/modules/loadDemo.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  try {
+    demoInDisk.forEach((to, index) => {
+      const from = pathToStartDemo[index];
+      copyFileSync(from, to);
+    });
+  } catch (_) {
+    /* we do not handle this */
+  }
+
+  return demoInDisk;
```
Comment (cpvalente) [inline apps/server/src/modules/loadDemo.ts:23]: question: what does the return do? it seems mostly unused?

File: apps/server/src/setup.ts [hunk 2]
```diff
@@ -80,11 +80,26 @@ export const pathToStartDb = isTest
   ? join(currentDirectory, '../', config.database.testdb, config.database.filename)
   : join(currentDirectory, '/preloaded-db/', config.database.filename);
 
+//TODO: move all to external
```
Comment (cpvalente) [inline apps/server/src/setup.ts:83]: suggestion: can you clarify this comment for future us? what is all?
Would the note be TO`DO: move all static files to the external directory?`

File: apps/server/src/modules/loadDemo.ts (hunk 1 above)
Comment (alex-Arc) [reply apps/server/src/modules/loadDemo.ts:23]: yes I don't know 
I just kept the same format as the other loader, bu i can remove it from them both?

Comment (cpvalente) [reply apps/server/src/modules/loadDemo.ts:23]: I guess if they are not used we should remove it

**Review by cpvalente — APPROVED** [review]
good stuff 🚀

**Pull request merged**